src/components/labeledInput.tsx

```diff
@@ -0,0 +1,271 @@
+import React, { ReactElement } from 'react'
+import { TextField, Box, Checkbox, FormControlLabel } from '@material-ui/core'
+import { Control, Controller, RegisterOptions } from 'react-hook-form'
+import InputMask from 'react-input-mask'
+import locationPostalCodes from '../data/locationPostalCodes'
+
+interface Errors {
+  [key: string]: {type: string}
+}
+
+type Register = (rules?: RegisterOptions) => any
+
+interface PatternConfig {
```

Comment:
Could these interfaces be factored into a separate file?

Reply:
:recycle: